### ISSUE TITLE
Policy files missing from this project

### ISSUE THREAD

**Issue by tarunkhurana1** [yoelhor/Azure-AD-B2C-UserMigration#1]
The article "https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-user-migration#optional-download-the-complete-policy-files" refers to 

"Step 4.3: Add a technical profile and technical profile validation to your policy
In Solution Explorer, expand "Solution Items", and open the TrustFrameworkExtensions.xml policy file."

but there are no policy files. Can you please look into it.

**Comment by leniel** [comment]
Yep... this is not present and the documentation part that deals with the policies is lacking more detailed steps.

https://docs.microsoft.com/en-us/azure/active-directory-b2c/active-directory-b2c-user-migration#step-43-add-a-technical-profile-and-technical-profile-validation-to-your-policy
